pytest.ini

```diff
@@ -6,7 +6,7 @@ log_date_format = %Y-%m-%d %H:%M:%S
 python_files = tests/*.py tests/**/*.py
 testpaths = tests
 norecursedirs = tests/api
-timeout = 300
+timeout = 600
```

Comment:
Was this to pass Travis CI? If so, probably just remove it as Nadia will attempt to fix tests failing on Travis.